### PR TITLE
Update yaml for Sen Dianne Feinstein (VAN-49252)

### PR DESCRIPTION
**Pull request by OdeclasV**
Feinstein's yaml was updated with new selectors and a topic variable that is required in her form. Wait values (initial and last) were also increased to account for a long waiting time in Feinstein's form to send a Thank You message when completing the form. 

These changes relate to ticket VAN-49252.

### CODE REVIEW

File: members/F000062.yaml [hunk 1]
```diff
@@ -5,40 +5,48 @@ contact_form:
...
-          selector: "#field_289B92DD-13E8-40E7-B141-1A02C6ACDCEA"
+          selector: input#field_289B92DD-13E8-40E7-B141-1A02C6ACDCEA
           value: $ADDRESS_STREET_2
           required: false
         - name: city
-          selector: "#field_C44B713C-04AD-411A-9744-6E7F2FF71898"
+          selector: input#field_C44B713C-04AD-411A-9744-6E7F2FF71898
           value: $ADDRESS_CITY
           required: true
+        - name: state
```
Comment (annanhan) [inline members/F000062.yaml]: you don't need this. the state is already pre-filled on the form. also, does this work? the state field is a dropdown.

File: members/F000062.yaml [hunk 2]
```diff
@@ -5,40 +5,48 @@ contact_form:
...
-          selector: "#field_64D2124B-A227-4F90-AD83-65DBC8175A69"
+          selector: input#field_64D2124B-A227-4F90-AD83-65DBC8175A69
           value: $ADDRESS_ZIP5
           required: true
         - name: email
-          selector: "#field_D065BDD9-7C1E-4D99-A50E-6A1ADAE8803B"
+          selector: input#field_D065BDD9-7C1E-4D99-A50E-6A1ADAE8803B
           value: $EMAIL
           required: true
+        - name: topic
```
Comment (annanhan) [inline members/F000062.yaml]: you don't need this either. The topic is getting selected here: https://github.com/NGPVAN/contact-congress/pull/358/files#diff-8f60da39bd935d35154e83adcc712abdR55

Comment (OdeclasV) [reply members/F000062.yaml]: got it. Will take this part out.

File: members/F000062.yaml (hunk 1 above)
Comment (OdeclasV) [reply members/F000062.yaml]: Good call, didn't notice that. Will take it out

**Pull request merged**